python/dgl/graphbolt/minibatch_sampler.py

```diff
@@ -1,27 +1,19 @@
 """Minibatch Sampler"""
 
-from typing import Mapping, Optional
+from collections.abc import Mapping
+from functools import partial
+from typing import Optional
 
 from torch.utils.data import default_collate
 from torchdata.datapipes.iter import IterableWrapper, IterDataPipe
 
 from ..batch import batch as dgl_batch
 from ..heterograph import DGLGraph
-from .itemset import ItemSet
+from .itemset import DictItemSet, ItemSet
```

Comment:
Small suggestion: perhaps rename `DictItemSet` to `ItemSetDict` because the former is not a subclass of `ItemSet`.

Reply:
pls let me change it in another PR to be clear.